rust/matches/src/lib.rs

```diff
@@ -95,6 +99,41 @@ pub mod matches {
 
     use super::*;
 
+    pub fn resize_oracle<'a, 'b, 'c, 'info>(
+        ctx: Context<'a, 'b, 'c, 'info, ResizeOracle<'info>>,
+        args: ResizeOracleArgs,
+    ) -> Result<()> {
+          
+        let win_oracle = &mut ctx.accounts.winOracle;
+
+        let win_oracle_account = win_oracle.to_account_info();
+        
+        if args.resize as usize > win_oracle_account.data.borrow().len() {
+            let system_program = &ctx.accounts.system_program;
+            let payer = &ctx.accounts.payer;
```

Comment:
You're not doing anything with this intermediate variable. Remove it and update the line below to:
```
let payer_account = &ctx.accounts.payer.to_account_info();
```
or
```
let payer = &ctx.accounts.payer.to_account_info();
```